cfgov/ask_cfpb/models/pages.py

```diff
@@ -86,97 +56,44 @@ def get_reusable_text_snippet(snippet_title):
 
 def get_ask_nav_items(request, current_page):
     from ask_cfpb.models import Category
-    return [
-        {
-            'title': cat.name,
-            'url': '/ask-cfpb/category-' + cat.slug + '/',
+    items = []
+    for cat in Category.objects.all():
+        if current_page.language == 'es':
+            title = cat.name_es
+            url = '/es/obtener-respuestas/categoria-{}/'.format(cat.slug_es)
```

Comment:
Would it be possible to define `/es/obtener-respuestas/` and `/ask-cfpb/` in a setting somewhere, since it's referenced quite a bit in the code?   I know this particular piece of code will go away soon once we deprecate category pages, but just for all the other references as well.

Reply:
@richaagarwal I think that definitely makes sense. Should we add that in this PR, or in a future one when we do some cleanup in this file?